manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -56,6 +56,8 @@ spec:
           value: "quay.io/openshift/origin-multus-whereabouts-ipam-cni:latest"
         - name: ROUTE_OVERRRIDE_CNI_IMAGE
           value: "quay.io/openshift/origin-multus-route-override-cni:latest"
+        - name: MULTUS_NETWORKPOLICY_IMAGE
```

Comment:
You need to add this to `manifests/image-references`

Reply:
Changed.